install.sh

```diff
@@ -88,76 +81,100 @@ fi
 
 read -p "Server MySQL openvpn-admin user password: " -s mysql_pass; echo
 
-
 # TODO MySQL port & host ?
 
 
 printf "\n################## Certificates informations ##################\n"
-key_size="0"
 
-while [ "$key_size" != "1024" -a "$key_size" != "2048" -a "$key_size" != "4096" ]; do
-  read -p "Key size (1024, 2048 or 4096): " key_size
-done
+read -p "Key size (1024, 2048 or 4096) [2048]: " key_size
```

Comment:
Because with EasyRSA 3 there is a default value, which is 2048, and the loop doesn't allow to just keep it by pressing Enter.